src/pykeen/experiments/validate.py

```diff
@@ -153,37 +153,37 @@ def _check(
         return value
 
     _check(
```

Comment:
It looks like you could also make `_check` a method of the `Resolver`.

Reply:
I will look into this

Reply:
I looked into this and it's a bit more complicated since it's playing around with local state. I will punt this one for next time ;)